doc/old_and_new_proposal.md

```diff
@@ -108,18 +178,304 @@ addition, every volume gets a "weight" (so far, based on the
+and also the behavior of the expert partitioner regarding them.
+
+It will be basically a collection of `volume` subsections, each of them with the
+options listed here. Having read the "How the new proposal distributes the
+space" may be important to fully understand some of them.
+
+  * `mount_point`. Directory where the volume will be mounted in the system.
+  * `fstypes`. A collection of acceptable file system types. If no list if
+    given, YaST will use a fallback one based on the mount point.
+  * `default_fstype`. Default file system type to format the volume.
+  * `desired_size`. Initial size to use in the first proposal attempt.
+  * `min_size`. Initial size to use in the second proposal attempt.
+  * `max_size`. Maximum size to assign to the volume. It can also contain the
+    value "unlimited" (meaning as big as possible). This will be considered the
+    default value if the option is not present.
+  * `max_size_lvm`. When LVM is used, this option can be used to override the
+    value at `max_size`.
+  * `weight`. Value used to distribute the extra space (after assigning the
+    minimal ones) among the volumes.
+  * `adjust_by_ram`. Whether the initial and max sizes of each attempt should
+    be adjusted based in the RAM size. Possible values are "true", "false",
+    "optional", with "false" being the default if not present and "optional"
+    meaning the user can decide via UI. So far the adaptation consist in
+    ensuring all the sizes are, at least, as big as the RAM. In the future, an
+    extra `adjust_by_ram_mode` option could be added to allow other approaches.
+  * `default_adjust_by_ram`. If the previous value is "optional", whether to
+    enable the option by default.
+  * `optional`. If true, the user will be able to skip the creation of this
+    volume.
+  * `default_enabled`. If the volume is optional, whether its creation should be
+    enabled by default.
+  * `fallback_for_min_size`. Mount point of another volume. If the volume being
+    defined is disabled, the `min_size` of that another volume will be increased
+    by the `min_size` of this disabled volume.
+  * `fallback_for_desired_size`. Same than before, but for `desired_size`.
+  * `fallback_for_max_size`. Same than before, but for `max_size`.
+  * `fallback_for_max_size_lvm`. Same than before, but for `max_size_lvm`.
+  * `fallback_for_weight`. Same than before, but for the volume weight.
+
+Some options will only apply if the chosen filesystem type for the volume is
+Btrfs, in some cases with the same name and meaning that in the old
+`control.xml` format. The main difference is that now the setting will apply to
+the volume in which it's included, not necessarily to the root ("/") one. In the
+expert partitioner, if a Btrfs filesystem is created and assigned to the mount
+point of the volume, these settings will also be used to suggest the filesystem
+options.
+
+  * `snapshots`. Whether snapshots should be activated. Possible values are
```

Comment:
Same than before. Maybe it would be clearer to have `snaphsots` and `optional_snapshots`.